build.sbt

```diff
@@ -34,16 +22,13 @@ lazy val commonSettings = Seq(
   name := "firrtl",
   version := "1.4-SNAPSHOT",
   scalaVersion := "2.12.11",
-  crossScalaVersions := Seq("2.12.11", "2.11.12"),
-  addCompilerPlugin(scalafixSemanticdb),
```

Comment:
Why do we remove `scalafixSemanticdb`? I think it's a not related change.

Reply:
There wasn't a `semantic-db` package for `2.13`. We should re-add scalafix before merging this PR, but I do not know how to.

Reply:
fixed in https://github.com/freechipsproject/firrtl/pull/1796/commits/9250cd688abb482a9c184f3cbf3bda172ac47050

Reply:
For the record I think we just need to bump the version in `project/plugins.sbt`: https://scalacenter.github.io/scalafix/docs/users/installation.html

Reply:
yeah, see https://github.com/freechipsproject/firrtl/pull/1796/commits/9250cd688abb482a9c184f3cbf3bda172ac47050